src/components/App.js

```diff
@@ -35,9 +29,21 @@ export default function App() {
         <Routes>
           <Route path="/" element={<SignIn />} />
           <Route path="/sign-up" element={<SignUp />} />
-          <Route path="/timeline" element={<Timeline />} />
-          <Route path="/hashtag/:hashtag" element={<Hashtag />} />
-          <Route path="/user/:id" element={<UserTimeline />} />
+          <Route path="/timeline" element={
+            <PrivatePage>
+                <Timeline />
+            </PrivatePage>
+          } />
```

Comment:
**Suggestion:** I believe that the code in the routes part is cleaner if you encompass this page before setting the element
```
const PrivateTimeLine =  <PrivatePage> <Timeline /> </PrivatePage>
```
```suggestion
           <PrivateTimeLine/> } />
```